src/Symfony/Component/AssetMapper/Command/DebugAssetMapperCommand.php

```diff
@@ -40,10 +42,41 @@ public function __construct(
     protected function configure(): void
     {
         $this
+            ->addArgument('name', InputArgument::OPTIONAL, 'An asset name (or a path) to search for (e.g. "app")')
+            ->addOption('ext', null, InputArgument::OPTIONAL, 'Filter assets by extension (e.g. "css")', null, ['js', 'css', 'png'])
```

Comment:
```suggestion
            ->addOption('ext', null, InputOption::VALUE_REQUIRED, 'Filter assets by extension (e.g. "css")', null, ['js', 'css', 'png'])
```

Just `--ext` is invalid, correct?

Reply:
Absolutely, thank you!